### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by Lockless404**
Completed the desktop version of the Portfolio website.

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -763,3 +749,351 @@ textarea {
   position: sticky;
   right: 35px;
 }
+
+@media (min-width: 600px) {
```
Comment (Mihndim2020) [inline portfolio.css:753]: - [ ] Please fix the media queries for the Desktop breakpoint, the value for min-width property should be in range of 992px - 1100px.

File: portfolio.html [hunk 2]
```diff
@@ -17,8 +17,8 @@
     </nav>
   </header>
```
Comment (Mihndim2020) [inline portfolio.html:18]: - [ ] Please make sure you desktop looks same as in the design, the desktop version has links to the various sections of the page, and the hamburger button is removed. Kindly fix that. See image below. 

***The design***
lockD1

***Your implementation***
locki1

File: portfolio.html [hunk 3]
```diff
@@ -165,7 +165,7 @@ <h1 class="about-me">
     <form action="https://formspree.io/f/xvodlvrj" method="POST">
       <input type="text" name="name" id="name" placeholder="Full name" class="input-name" required minlength="1" maxlength="30"> <br>
       <input type="email" id="email" placeholder="Email address" class="input-email" required minlength="1"><br>
-      <textarea name="text" minlength="1" required maxlength="500">Enter text here</textarea>
+      <textarea name="text" class="inputText" minlength="1" required maxlength="500">Enter text here</textarea>
       <button class="touch-button" type="submit">Get in touch</button>
     </form>
```
Comment (Mihndim2020) [inline portfolio.html:170]: - [ ] This components in this section are overlapping, the about me text is overlapping with the form, please kindly fix that. See images below. 

***The design***
lockd2

***Your implementation**
locki2
*

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Lockless404, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Project is running
:heavy_check_mark: Linters are passing

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Requires Changes ♻️
_Check the comments under the review._

Please also consider:
- [ ] updating to readme file to reflect the current changes, i.e the desktop version.
- [ ] removing sub headings where information in not available, like prerequisites, setup etc...
- [ ] fixing your screenshot link


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please also consider:
- [OPTIONAL] hosting the site and adding a live link in your readme file.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @Lockless404,**

### **Project Approved ✅** 

Approved


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**